controller/cartController.js

```diff
@@ -0,0 +1,113 @@
+import { BaseController } from "./baseController";
+const stripe = require('stripe')(process.env.STRIPE_SECRET_KEY);
+
+Object.filter = (obj, predicate) => 
+    Object.keys(obj)
+          .filter( key => predicate(obj[key]) )
+          .reduce( (res, key) => (res[key] = obj[key], res), {} );
+
+export class CartController extends BaseController {
+    constructor() {
+        this.bag = localStorage.getItem('bag') ? localStorage.getItem('bag') : {};
+        this.total = localStorage.getItem('price') ? localStorage.getItem('price') : 0;
+    }
+
+    updateLocalStorage() {
+        localStorage.setItem('price', Object.keys(this.bag).reduce((acc, key) => acc + this.bag[key].price * this.bag[key].amount, 0));
+        localStorage.setItem('bag', this.bag);
+    }
+     // Implement methods specific to the shopping bag controller here
+    update(req, res) {
```

Comment:
so luong nua, nghi cach khac hoac implement 1 shopping cart class de handle cac case

Reply:
so luong what?

Reply:
so luong cua do chu sao, t map dit t an 2 cai burger

Reply:
this.bag = {
item.id : {item.price, item.amount}
}